### PR TITLE
Solidity Coverage - Signature example

### LINKED ISSUE #492: [Solidity Support] Signature example
### Problem

The Hedera network supports solidity as one of the smart contracts development languages.
Occasionally items have shown up that surfaced in-equivalence at the language layer.

### Solution

Implement [Signaure creating and verification example](https://docs.soliditylang.org/en/latest/solidity-by-example.html#creating-and-verifying-signatures) and add test to cover all methods

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
- Adding test coverage for signature example and its tests, 
- Also fixed a typo on TypeOps tests

**Related issue(s)**:

Fixes #492 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: contracts/solidity/signature-example/ReceiverPays.sol [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.7.0 <0.9.0;
+// This will report a warning due to deprecated selfdestruct
```
Comment (Nana-EC) [inline contracts/solidity/signature-example/ReceiverPays.sol:3]: Q: should this comment move to the shutdown method?

File: test/constants.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ const Path = {
   HIP583_ERC721Mock: 'contracts/hip-583/ERC721Mock.sol:ERC721Mock',
   HRC: 'contracts/hrc/HRC.sol:HRC',
   TYPE_OPS: 'contracts/solidity/typeops/TypeOps.sol:TypeOps',
+  RECEIVER_PAYS: 'contracts/solidity/signature-example/ReceiverPays.sol:ReceiverPays',
```
Comment (Nana-EC) [inline test/constants.js:69]: Q: Should we simplify it to signature?

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looks good.
2 suggestions

File: test/constants.js (hunk 2 above)
Comment (AlfredoG87) [reply test/constants.js:69]: I based myself on the names of the contracts described on the example provided with the issue.
https://docs.soliditylang.org/en/latest/solidity-by-example.html#creating-and-verifying-signatures 

But I can make the refactor nonetheless. 👍

File: contracts/solidity/signature-example/ReceiverPays.sol (hunk 1 above)
Comment (AlfredoG87) [reply contracts/solidity/signature-example/ReceiverPays.sol]: ✅  Done

File: contracts/solidity/signature-example/ReceiverPays.sol [hunk 3]
```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0
```
Comment (Nana-EC) [inline contracts/solidity/signature-example/ReceiverPays.sol:1]: Use Apache

**Review by Nana-EC — DISMISSED** [review]
LG
Just fix the license on the contract

**Review by ebadiere — DISMISSED** [review]
LG.

**Review by Nana-EC — DISMISSED** [review]
LG

**Review by ebadiere — APPROVED** [review]
LG.

**Pull request merged**